packages/commerce/src/api/utils/index.ts

```diff
@@ -44,7 +44,9 @@ export const transformRequest = (req: NextApiRequest, path: string) => {
     body = JSON.stringify(req.body)
   }
 
-  return new NextRequest(`https://${req.headers.host}/api/commerce/${path}`, {
+  const url = new URL(req.url || '/', `https://${req.headers.host}`)
+
+  return new NextRequest(url, {
```

Comment:
The `path` parameter of this function is no longer used after this change, why was this changed? not mentioned in the description 🤔

Reply:
`req.url` comes with full path + query search, which was lacking before.